api/views/personal_info.py

```diff
@@ -76,3 +82,30 @@ def check_existence_of_chat_id(self, request: Request) -> Response:
         serializer = self.get_serializer(data=request.query_params)
         serializer.is_valid(raise_exception=True)
         return Response(request.query_params)
+
+    @extend_schema(
+        responses={
+            status.HTTP_200_OK: OpenApiResponse(
+                response=PersonalInfoSerializer,
+                description="Users with these parameters exist",
+            ),
+            status.HTTP_406_NOT_ACCEPTABLE: OpenApiResponse(
+                response=BaseAPIExceptionSerializer,
+                description="No user with these parameters exists",
+            ),
+        }
+    )
+    def list(
```

Comment:
I would keep an empty list because, on the user side, you need to create an explicit error handler for 406

Reply:
Btw, maybe it is already working that way but, in general, middleware is a good idea, it is usually a good practice to handle all your exceptions (for all codes) in a single place and pass whatever parameter you need in the Exception itself

Reply:
> I would keep an empty list because, on the user side, you need to create an explicit error handler for 406

Keeping an empty list as voted by the dev team